app/src/main/kotlin/com/wire/android/ui/home/settings/backup/BackupAndRestoreViewModel.kt

```diff
@@ -41,7 +43,8 @@ class BackupAndRestoreViewModel
     private val createBackupFile: CreateBackupUseCase,
     private val verifyBackup: VerifyBackupUseCase,
     private val kaliumFileSystem: KaliumFileSystem,
-    private val fileManager: FileManager
+    private val fileManager: FileManager,
+    private val dispatchers: DispatcherProvider,
```

Comment:
I think we are not using this injected `dispatchers` field ? Are we expecting to use it, otherwise I believe we can remove it 🗑️

Reply:
Sorry I totally forgot to use it for the cancellation. Updated it.